pyodide-build/pyodide_build/_f2c_fixes.py

```diff
@@ -478,8 +493,14 @@ def scipy_fix_cfile(path: str) -> None:
         text = text.replace(",size_t", "")
         text = re.sub(r",slen\([a-z]*\)\)", ")", text)
 
-    if path.endswith("_fblasmodule.c"):
-        text = text.replace(" float (*f2py_func)", " double (*f2py_func)")
+    if path.endswith("stats/statlib/spearman.c"):
+        # in scipy/stats/statlib/swilk.f ALNORM is called with a double, and in
+        # scipy/stats/statlib/spearman.f with a real this generates
+        # inconsistent signature. Let's use double in both, I don't think this
+        # code path will work (but at least it will compile) since it needs
+        # "ALNORM = algorithm AS66", which I don't think we have with the f2c
+        # route
+        text = text.replace("extern real alnorm_", "extern doublereal alnorm_")
```

Comment:
Is there an upstream bug here?

Reply:
Good point indeed, probably worth a PR to scipy